elementary/monitor/api/sidebar/sidebar.py

```diff
@@ -48,3 +68,33 @@ def _update_dbt_sidebar(
                 if part not in dbt_sidebar:
                     dbt_sidebar[part] = {}
                 dbt_sidebar = dbt_sidebar[part]
+
+    def get_tags_sidebar(
+        self,
+        models: Dict[str, NormalizedModelSchema],
+        sources: Dict[str, NormalizedModelSchema],
+    ) -> TagsSidebarSchema:
```

Comment:
Maybe we can somehow merge these two functions?
E.g. `get_sidebar_by_labels` or something

(Not very critical, as these are very short functions)

Reply:
I like the seperation because this is an equivalent to an API.
I can make a common method that is used by both.

Reply:
Not critical, just my automatic duplication radar :)
I think it's also fine as-is as it's very clear

Reply:
That is a good radar!